.github/workflows/continuous-integration-workflow.yml

```diff
@@ -82,6 +95,7 @@ jobs:
                 ct-ng olddefconfig; \
                 ct-ng source; \
           done
+          find src -type f -not -newer stamp -delete -print
           tar -cvf src.tar src
```

Comment:
Do we have to do anything to tell the cache to be updated with what is now in src.tar?

Reply:
to clarify ^: there are `actions/cache/restore`, `actions/cache/save` and `action/cache` with combines the fomer 2